builder/amazon/common/access_config.go

```diff
@@ -68,25 +68,27 @@ func (c *AccessConfig) Session() (*session.Session, error) {
 		SharedConfigState: session.SharedConfigEnable,
 		Config:            *config,
 	}
+
 	if c.MFACode != "" {
 		opts.AssumeRoleTokenProvider = func() (string, error) {
 			return c.MFACode, nil
 		}
 	}
-	var err error
-	c.session, err = session.NewSessionWithOptions(opts)
-	if err != nil {
+
+	if session, err := session.NewSessionWithOptions(opts); err != nil {
```

Comment:
Can we use a name other than session here?  That namespace is really full.

Reply:
oh yeah, that shadows the package, good catch